src/pseudoflow/core/libhpf.c

```diff
@@ -1709,16 +1808,18 @@ initializeParametricCut - Set up data structures for parametric cut
     }
 
     /* initialize problem for LAMBDA_LOW */
-    initializeContractedProblem(lowProblem, nodeListSuper, numNodesSuper, arcListSuper, numArcsSuper,LAMBDA_LOW, all_sink, all_source);
+    initializeContractedProblem(lowProblem, nodeListSuper, numNodesSuper,
+			arcListSuper, numArcsSuper,LAMBDA_LOW, all_sink, all_source, 0);
 
 	if (useParametricCut == 1)
 	{
 		/* initialize problem for LAMBDA_HIGH */
-		initializeContractedProblem(highProblem, nodeListSuper, numNodesSuper, arcListSuper, numArcsSuper,LAMBDA_HIGH, all_sink, all_source);
+		initializeContractedProblem(highProblem, nodeListSuper, numNodesSuper,
+			arcListSuper, numArcsSuper,LAMBDA_HIGH, all_sink, all_source, 1);
 	}
 
-    free(all_sink);
-    free(all_source);
+    //free(all_sink);
```

Comment:
Please delete unnecessary commented out code. Debug help is fine (although in hindsight, https://stackoverflow.com/questions/1644868/define-macro-for-debug-printing-in-c is better)